views/pulsar/v2/helpers/form.html.twig

```diff
@@ -859,7 +862,7 @@ Option      | Type   | Description
 autofocus   | bool   | Whether the field should have input focus on page load
 disabled    | bool   | Stops the element from being interactive if true
 form        | string | Specific one or more forms this label belongs to
-format      | string | Options for this can be `default`, `US` or `reverse`
+data-format | string | Options for this can be `default`, `US` or `reverse`
```

Comment:
Checked XFP and the `format` attribute is in use, may need to switch back to `format` and convert that to `data-format` inside the helper.

Reply:
Reverted the comment change. The `format` option is still used to set the format, this is passed to `data-format` within the help. The key change here is that `format` is excluded from the markup (which was an invalid attribute).